.github/workflows/build_ubuntu.yml

```diff
@@ -0,0 +1,120 @@
+      - develop
+  pull_request:
+  release:
+    types: [ created ]
+
+env:
+  GITHUB_TOKEN: ${{ github.token }}
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        xprs: [XPRESS-ON, XPRESS-OFF]
+    env:
+      XPRESSDIR: ${{ github.workspace }}/xpress
+      XPRESS: ${{ github.workspace }}/xpress/bin
+      XPRS_LIB_Path: ${{ github.workspace }}/xpress/lib
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Checkout xpressmp linux
+        if: matrix.xprs == 'XPRESS-ON'
+        uses: actions/checkout@v2
+        with:
+          repository: rte-france/xpress-mp
+          path: ${{ env.XPRESSDIR }}
+          ref: master
+          token: ${{ secrets.AS_TOKEN }} #reniew token periodically
+
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        with:
+          key: ${{ matrix.os }}-${{ matrix.xprs }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          cache: 'pip'
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-tests.txt
+          pip install -r requirements-ui.txt
+
+      - name: Install mandatory system libraries
+        run: |
+          sudo apt-get update --fix-missing
+          sudo apt-get install libjsoncpp-dev libgtest-dev libboost-mpi-dev libboost-program-options-dev libtbb-dev
+          cd /usr/src/googletest/
+          sudo cmake .
+          sudo cmake --build .
+
+      - name: Read antares-solver version
+        id: antares-version
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: 'antares-version.json'
+          prop_path: 'antares_version'
+
+      - name: Read antares-xpansion version
+        id: antares-xpansion-version
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: 'antares-version.json'
+          prop_path: 'antares_xpansion_version'
+
+      - name: Read antares-deps version
+        id: antares-deps-version
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: 'antares-version.json'
+          prop_path: 'antares_deps_version'
+
+      - name: Download pre-compiled librairies
+        uses: ./.github/workflows/download-extract-precompiled-libraries-tgz
+        with:
+          antares-deps-version: ${{steps.antares-deps-version.outputs.prop}}
+          antares-version: ${{steps.antares-version.outputs.prop}}
+          os: ${{matrix.os}}
+          os-full-name: Ubuntu-20.04
+
+      - name: Configure
+        shell: bash
+        #XPRESS-VALUE = ${{ matrix.xprs }} == "XPRESS-ON" ? "ON" : "OFF"
+        run: |
+          [[ ${{ matrix.xprs }} == "XPRESS-ON" ]] && XPRESS-VALUE="ON" || XPRESS-VALUE="OFF" 
```

Comment:
This line makes the job fail
```
/home/runner/work/_temp/ff2ac76a-f83b-460f-9f55-751de179eb48.sh: line 1: XPRESS-VALUE=OFF: command not found
```

Reply:
Fixed. A bash variable can't contain the dash "-" character, I used undescores "_" instead